### ISSUE TITLE
Add automation to mirror repo to second server

### ISSUE THREAD

**Issue by VuiMuich** [leftwm/leftwm#1053]
Recently there has been expressed the desire on discord, to have the repo (and possibly the sister like `community-themes`, `lefthk`, `leftwm-theme` etc. repos as well) mirrored to another provider, as github seems to be difficult to reach from some contries (i.e. India).

I have started to do this by manually migrating the repos to codeberg.org but in the longrun some automation would be the only sensible way.

As I don't really have any experience in this area I thought it might be a good idea to have an issue tracking this.
So if anybody would have a suggestion, we are totally open to it.

**Comment by hertg** [comment]
Can you add me to the org you created on codeberg, I'm also @hertg there, thanks

**Comment by Eskaan** [comment]
> So if anybody would have a suggestion, we are totally open to it.

I also have no experience with this, but I think the go-to way would be a GitHub action that runs on push for main, that gets a secret access token supplied (I suppose codeberg has something like this too).

Then the usual 
```
git remote add mirror https://x-access-token:${{ secrets.TOKEN }}@https://codeberg.org/leftwm/leftwm.git
git push
```

**Comment by VuiMuich** [comment]
> Can you add me to the org you created on codeberg, I'm also @hertg there, thanks

Yup, definitely will. I just started this in a free minute and hadn't time to continue yet..

**Comment by hertg** [comment]
I took a quick glance at https://github.com/pixta-dev/repository-mirroring-action and it looks alright, very short and concise action with no convoluted commands, ie. no shady shenanigans. We could pin it against the latest commit I just checked. 

```yaml
- uses: pixta-dev/repository-mirroring-action@674e65a7d483ca28dafaacba0d07351bdcc8bd75
```

Maybe it would also be wise to create a leftbot account on codeberg, and then store its credentials in our 1password. This way we won't have to add any of our personal codeberg ssh keys to the CI.
